File: app/src/main/java/com/platdmit/simplecloudmanager/data/database/DbManager.java
package com.platdmit.simplecloudmanager.data.database;

import com.platdmit.simplecloudmanager.data.database.dao.*;
import com.platdmit.simplecloudmanager.data.database.entity.*;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {DbAccount.class, DbDomain.class, DbDomainRecord.class,
        DbServer.class, DbAction.class, DbBackup.class, DbLoadAverage.class,
        DbStatistic.class, DbUpdateSchedule.class}, version = 1, exportSchema = false)
public abstract class DbManager extends RoomDatabase {
    public abstract AccountDao mAccountDao();
    public abstract DomainDao mDomainDao();
    public abstract DomainRecordDao mDomainRecordDao();
    public abstract ServerDao mServerDao();
    public abstract ActionDao mActionDao();
    public abstract BackupDao mBackupDao();
    public abstract LoadAverageDao mLoadAverageDao();
    public abstract StatisticDao mStatisticDao();
    public abstract UpdateScheduleDao mUpdateScheduleDao();
}

File: app/src/main/java/com/platdmit/simplecloudmanager/data/database/DbManager.kt
package com.platdmit.simplecloudmanager.data.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.platdmit.simplecloudmanager.data.database.dao.*
import com.platdmit.simplecloudmanager.data.database.entity.*

@Database(entities = [
    DbAccount::class, DbDomain::class, DbDomainRecord::class, DbServer::class,
    DbAction::class, DbBackup::class, DbLoadAverage::class, DbStatistic::class, DbUpdateSchedule::class
], version = 1, exportSchema = false)
abstract class DbManager : RoomDatabase() {
    abstract fun mAccountDao(): AccountDao
    abstract fun mDomainDao(): DomainDao
    abstract fun mDomainRecordDao(): DomainRecordDao
    abstract fun mServerDao(): ServerDao
    abstract fun mActionDao(): ActionDao
    abstract fun mBackupDao(): BackupDao
    abstract fun mLoadAverageDao(): LoadAverageDao
    abstract fun mStatisticDao(): StatisticDao
    abstract fun mUpdateScheduleDao(): UpdateScheduleDao
}